src/app/constants/endpoints.ts

```diff
@@ -13,3 +13,8 @@ export const endpoints = {
 export const ftxEndpoints = {
   baseUrl: 'https://ftx.us/api/',
 };
+
+export const zendeskEndpoints = {
+  baseUrl: 'https://dolphinentertainment.zendesk.com/api/v2/',
```

Comment:
WDYT about moving those values to env variables?

Reply:
good Idea, will do it. Thanks

Reply:
I will merge this PR and create a new one with all the changes to use env variables.